### PR TITLE
Feature/cpb-49144

### PR DESCRIPTION
**Pull request by wangmingf2019**
1. Enhance for source code, ensure that installer is normally work for not branchname, chart path and namespace;
2. Add new addon.

### CODE REVIEW

File: VERSIONINFO [hunk 1]
```diff
@@ -1,2 +1,2 @@
-appVersion=0.1.1
-version=0.1.1
\ No newline at end of file
+appVersion=0.1.3
```
Comment (MingZhang-PS) [inline VERSIONINFO:1]: why the appVersion is not 0.1.2?

Comment (wangmingf2019) [reply VERSIONINFO:1]: In docker registry, the version 0.1.2 is existing. 
So I don't want to cover it.

File: backend/src/installerservice/installerservice.service.ts [hunk 2]
```diff
@@ -189,4 +188,20 @@ export class InstallerService {
             return (accessUrl && accessUrl !== '') ? 'INSTALLED' : 'INSTALL_FAILED';
         }
     }
+
+    private async getHelmChartsPath(rootPath: string, chartConfigDataPath: string) {
+        let helmChartPath: string;
+        if (chartConfigDataPath) {
+            helmChartPath = rootPath + '/' + chartConfigDataPath;
+        } else {
+            const results = search.recursiveSearchSync('Chart.yaml', rootPath);
```
Comment (MingZhang-PS) [inline backend/src/installerservice/installerservice.service.ts:197]: maybe we can change it to an async call

File: VERSIONINFO (hunk 1 above)
Comment (wangmingf2019) [reply VERSIONINFO]: Done, update appversion to 0.1.2

File: backend/src/installerservice/installerservice.service.ts (hunk 2 above)
Comment (wangmingf2019) [reply backend/src/installerservice/installerservice.service.ts:197]: Recommend to using sync version for search single file.

**Pull request merged**